lib/jobs/ssh-job.js

```diff
@@ -37,6 +37,7 @@ function sshJobFactory(
         this.nodeId = this.context.target;
         this.commandUtil = new CommandUtil(this.nodeId);
         this.commands = this.commandUtil.buildCommands(options.commands);
+        this.sshExecOptions = options.sshExecOptions;
```

Comment:
Sorry, I misunderstand it, this is not ssh setting, but some arguments for ssh execution. please ignore my above comment.